src/countOccurrences/countOccurrences.test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const countOccurrences = require('./countOccurrences');
+
+describe('Testing countOccurrences', () => {
+  test('Should return 0 for: empty string', () => {
+    expect(countOccurrences('', 'a'))
+      .toBe(0);
+  });
+
+  test('Should return 0 for: "ba" in "samsung"', () => {
```

Comment:
Test description should explain what scenario is being tested, not samples of input/output. When we explain scenarios, we can where tests are duplicated. With samples of intput/output it's hard to tell whether all scenarios are being covered.

```suggestion
  test('Should return zero when substr not found in str.', () => {
```